package.json

```diff
@@ -79,8 +79,8 @@
     "json-stable-stringify": "^1.0.1",
     "lodash": "^4.17.4",
     "set-abi-gen": "1.1.0-beta.1",
-    "solc": "^0.4.25",
-    "solidity-coverage": "^0.5.11",
+    "solc": "^0.5.4",
+    "solidity-coverage": "https://github.com/leapdao/solidity-coverage#master",
```

Comment:
Any reason for changing to this?

Reply:
https://github.com/sc-forks/solidity-parser/pull/18 coverage was failing because of `_` in variable names and some other weird stuff that the current version of solidity parser doesn't handle still. we can maybe fork leapdaos too if we want to keep it under us for now